src/order/complete_lattice.lean

```diff
@@ -1295,6 +1295,54 @@ instance [has_Sup α] [has_Sup β] : has_Sup (α × β) :=
 instance [has_Inf α] [has_Inf β] : has_Inf (α × β) :=
 ⟨λ s, (Inf (prod.fst '' s), Inf (prod.snd '' s))⟩
 
+variables {α β}
+
+lemma fst_Inf [has_Inf α] [has_Inf β] (s : set (α × β)) : (Inf s).fst = Inf (prod.fst '' s) := rfl
+
+lemma snd_Inf [has_Inf α] [has_Inf β] (s : set (α × β)) : (Inf s).snd = Inf (prod.snd '' s) := rfl
+
+lemma swap_Inf [has_Inf α] [has_Inf β] (s : set (α × β)) : (Inf s).swap = Inf (prod.swap '' s) :=
```

Comment:
Oh I'm only noticing now, but maybe this statement is better?
```
lemma swap_Inf [has_Inf α] [has_Inf β] (s : set (α × β)) : (Inf s).swap = Inf (prod.swap ⁻¹' s) :=
```

Reply:
I don't really need this lemma anyway, I was just listing every trivial function it could interact with. Feel free to make a PR to spell it that way instead.